packages/atlas/src/config/contentFilter.ts

```diff
@@ -25,14 +17,11 @@ export const createVideoWhereObjectWithFilters = (
   return {
     ...videoWhereInput,
 
-    // filter by video ids
-    ...(filteredVideoIds.length ? { id_not_in: filteredVideoIds } : {}),
-
     // filter by channel ids
     channel: createChannelWhereObjectWithFilters(videoWhereInput?.channel),
 
     // filter by category
-    ...(!atlasConfig.content.showAllContent
+    ...(atlasConfig.content.showAllContent
```

Comment:
I would also remove `showAllContent` from the config. This option is also configurable in Orion. So I don't think we should keep it

Reply:
Also, let's remove `createVideoWhereObjectWithFilters` as well.